### PR TITLE
Feature/62 go tool release

### LINKED ISSUE #62: Add release process for go tools

### PR DESCRIPTION
**Pull request by sklein94**
Resolve #62

### CODE REVIEW

**Review by Subaro — APPROVED** [review]

File: build/make/release_functions.sh [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  git checkout develop
+  git pull origin develop
+  git flow release start v"${NEW_RELEASE_VERSION}"
+}
+
+update_versions(){
+  local NEW_RELEASE_VERSION="${1}"
+
+  if [ -f "dogu.json" ]; then
+    # Update version in dogu.json
```
Comment (Subaro) [inline build/make/release_functions.sh:78]: Move comments before if

File: build/make/release_functions.sh [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    jq ".Version = \"${NEW_RELEASE_VERSION}\"" dogu.json > dogu2.json && mv dogu2.json dogu.json
+  fi
+
+  if [ -f "Dockerfile" ]; then
+    # Update version in Dockerfile
```
Comment (Subaro) [inline build/make/release_functions.sh:83]: Move comments before if

File: build/make/release_functions.sh (hunk 1 above)
Comment (Subaro) [reply build/make/release_functions.sh:78]: add echo

File: build/make/release_functions.sh (hunk 2 above)
Comment (Subaro) [reply build/make/release_functions.sh:83]: add echo

File: build/make/release_functions.sh [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    sed -i "s/\(^[ ]*VERSION=\"\)\([^\"]*\)\(.*$\)/\1${NEW_RELEASE_VERSION}\3/" Dockerfile
+  fi
+
+  # Update version in Makefile
+  if [ -f "Makefile" ]; then
+    sed -i "s/\(^VERSION=\)\(.*\)$/\1${NEW_RELEASE_VERSION}/" Makefile
+  fi
+
+  # Update version in package.json
+  if [ -f "package.json" ]; then
```
Comment (Subaro) [inline build/make/release_functions.sh:93]: add echo

File: build/make/release_functions.sh [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+  fi
+
+  if [ -f "Dockerfile" ]; then
+    # Update version in Dockerfile
+    sed -i "s/\(^[ ]*VERSION=\"\)\([^\"]*\)\(.*$\)/\1${NEW_RELEASE_VERSION}\3/" Dockerfile
+  fi
+
+  # Update version in Makefile
+  if [ -f "Makefile" ]; then
+    sed -i "s/\(^VERSION=\)\(.*\)$/\1${NEW_RELEASE_VERSION}/" Makefile
```
Comment (Subaro) [inline build/make/release_functions.sh:89]: add echo

**Review by Subaro — CHANGES_REQUESTED** [review]

File: build/make/release_functions.sh (hunk 1 above)
Comment (sklein94) [reply build/make/release_functions.sh]: done

File: build/make/release_functions.sh (hunk 2 above)
Comment (sklein94) [reply build/make/release_functions.sh]: done

File: build/make/release_functions.sh (hunk 3 above)
Comment (sklein94) [reply build/make/release_functions.sh:96]: done

File: build/make/release_functions.sh (hunk 4 above)
Comment (sklein94) [reply build/make/release_functions.sh:92]: done

**Review by Subaro — APPROVED** [review]

**Pull request merged**